devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -1035,6 +1163,366 @@ def zwaveEvent(physicalgraph.zwave.commands.applicationstatusv1.ApplicationRejec
 	createEvent(displayed: true, descriptionText: "Rejected the last request")
 }
 
+/**
+ * Responsible for parsing ScheduleEntryTypeSupportedReport command
+ *
+ * @param cmd: The ScheduleEntryTypeSupportedReport command to be parsed
+ *
+ * @return The event(s) to be sent out
+ *
+ */
+def zwaveEvent(physicalgraph.zwave.commands.scheduleentrylockv3.ScheduleEntryTypeSupportedReport cmd) {
+	log.trace "[DTH] Executing 'zwaveEvent(physicalgraph.zwave.commands.scheduleentrylockv3.ScheduleEntryTypeSupportedReport cmd)' with cmd = $cmd"
+	def result = []
+	def numberOfSlotsDailyRepeating = cmd.numberOfSlotsDailyRepeating
```

Comment:
Can we do:

`def numberOfSlotsDailyRepeating = cmd.numberOfSlotsDailyRepeating ?: 0`

here?

Reply:
Or even just put that statement in the `descriptionText` directly?